packages/conf-zlib/conf-zlib.1/opam

```diff
@@ -18,7 +27,6 @@ depexts: [
   ["zlib"] {os-distribution = "homebrew" & os = "macos"}
   ["zlib"] {os-family = "arch"}
   ["zlib"] {os = "win32" & os-distribution = "cygwinports"}
-  ["zlib-devel"] {os-distribution = "cygwin"}
```

Comment:
This can be removed (see also #26130 - it should be `os = "cygwin"` rather than be deleted)